src/QMCDrivers/WFOpt/QMCCostFunctionBatched.cpp

```diff
@@ -436,6 +482,27 @@ void QMCCostFunctionBatched::checkConfigurations()
   app_log() << "  VMC Eavg = " << Etarget << std::endl;
   app_log() << "  VMC Evar = " << etemp[2] / etemp[1] - Etarget * Etarget << std::endl;
   app_log() << "  Total weights = " << etemp[1] << std::endl;
+
+ #ifdef HAVE_LMY_ENGINE
+  // engine finish taking samples
+  if (data.method == "adaptive")
+  {
+     //Only call sample_finish now if samples are not being stored
+    data.lmEngine->sample_finish();
+
+    if (data.lmEngine->block_first())
+    {
+      OptVariablesForPsi.setComputed();
+      app_log() << "calling setComputed function" << std::endl;
```

Comment:
Pointless output.  If its really helpful for debug I doubt its a permanent addition. If it is, this is a place for to use the preprocessor
```c++
#ifndef NDEBUG
// check concurrency environment if necessary
std::cout << "returned from setComputed\n";
#endif
```